recipes/cmake/3.x.x/conanfile.py

```diff
@@ -37,16 +38,16 @@ def requirements(self):
         if self.options.with_openssl:
             self.requires("openssl/1.1.1q")
 
-    def validate(self):
+    def validate_build(self):
```

Comment:
any reason for changing validate to validate_build?

Reply:
Hi @uilianries 
I am assuming my reply above was satisfactory and am therefore marking this issue "resolved"

Reply:
@System-Arch Please, revert. It should be validated for both consumer and when building. For instance, x86 is not supported. In case of consuming it, the x86 package will be missing and only will really fail when trying to build.

Reply:
Hi @uilianries,

I moved the mac x86 check back to the validate() method. I believe all of the other checks regarding compiler versions are indeed build-time checks and don't apply to simply using CMake as a build tool. Thanks,